File: control-flow/Q80/main.go
package main

import (
	"fmt"
)

func main() {
	f1 := 1
	f2 := 1
	var n int
	fmt.Print("Enter a number: ")
	fmt.Scan(&n)
	if n == 1 {
		fmt.Println(f1)
	} else if n == 2 {
		fmt.Println(f1)
		fmt.Println(f2)
	} else {
		fmt.Println(f1)
		fmt.Println(f2)
		var f3 int
		for i := 3; i <= n; i++ {
			f3 = f1 + f2
			fmt.Println(f3)
			f1 = f2
			f2 = f3
		}
	}
}

File: control-flow/Q80/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter a number:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: i32 = input.trim().parse().expect("Not a valid number");

    let mut f1: i32 = 1;
    let mut f2: i32 = 1;

    if n == 1 {
        println!("{}", f1);
    } else if n == 2 {
        println!("{}", f1);
        println!("{}", f2);
    } else {
        println!("{}", f1);
        println!("{}", f2);
        let mut f3: i32;
        for _ in 3..=n {
            f3 = f1 + f2;
            println!("{}", f3);
            f1 = f2;
            f2 = f3;
        }
    }
}